Observer/ProductDeleteBefore.php

```diff
@@ -54,8 +54,8 @@ public function __construct(
     public function execute(Observer $observer)
     {
         $product = $observer->getEvent()->getProduct();
-        $websiteIds = $product->getWebsiteIds();
-        $storeIdKlaviyoMap = $this->_klaviyoScopeSetting->getStoreIdKlaviyoAccountSetMap($websiteIds);
+        $storeIds = $product->getStoreIds();
```

Comment:
I could see downstream the storeIds is set inside the $data array.  Why wasnt it explicitly declared before? Am I missing something?

Reply:
this is a bug, we should have been getting storeIds of the product not websiteIds